doc/dev/config.rst

```diff
@@ -57,7 +57,7 @@ Reading configuration values
 There are two ways for Ceph code to get configuration values. One way is to
 read it directly from a variable named "g_conf," or equivalently,
 "g_ceph_ctx->_conf." The other is to register an observer that will be called
-every time the relevant configuration values changes.  This observer will be
+every time the relevant configuration values changes. This observer will be
```

Comment:
@tchaikov Yes, I completely accept your suggestion. But there were other significant changes in this PR along with these minors.